### DISCUSSION TITLE
What does this mean?

### DISCUSSION THREAD

**Discussion by a-wild-rickyb** [breezy-weather/breezy-weather#902] [Q&A] [answered]
When I look at the Sun and Moon tile the Sun and Moon are both showing at the same time and in the evening nothing is showing. So what does it mean? 
![Screenshot_20240407-135254](https://github.com/breezy-weather/breezy-weather/assets/130870205/48b3a0a3-b56c-44bb-9492-eecc4dd53c63)

**Comment by papjul** [comment]
Based on the data from your screenshot, sun rises at 6:49 AM and sets at 7:53 PM, and moon rises at 6:21 AM and sets at 6:46 PM.

**Reply by a-wild-rickyb** [reply]
But why is there the sun and moon at the same time, shouldn't there be the sun in the morning and the Moon in the night

**Reply by papjul** [reply]
Huh, no? The sun is all daytime, not just morning, and the moon can be seen during daytime or nighttime, it depends.

(Assuming you don't live near poles where the sun can be up 24 hours/24 or 0 hours/24)

**Reply by a-wild-rickyb** [reply]
Then how come the moon isn't showing at night

**Reply by papjul** [reply]
I'm sorry I don't understand where you're trying to get at? If the moon already showed during the whole daytime period, chances are very low to see it again during the night (it can happen but for simplicity, we only handle one moonrise per day), so it will be a moon-less night, just like you can have a moon-less day. The other hemisphere also needs its moon 😆

(Technically, the moon is showing during the night between 6:21 and 6:49)

**Reply by a-wild-rickyb** [reply]
But every single day the Moon is in the day and at night they're also is a moon so why isn't it showing in the app

**Reply by papjul** [reply, marked as answer]
Yes, the moon was visible that night, but only between 6:21 AM and 6:49 AM. After 6:49 AM, the sun rose so I believe we can no longer say it's the night. You have to wait for the next day to see the moon rise again at an approximately similar hour, which is why you no longer see the moon after 6:46 PM in the app, because the moon already rose once that day.

The moonrise and moonset is different all year. Sometimes it's only visible during the day, sometimes only the night, and most of the time, both, for example afternoon+evening, or night+morning.

If you have other astronomy questions, please ask on a specialized forum.
